plugins/woocommerce/includes/gateways/paypal/class-wc-gateway-paypal.php

```diff
@@ -527,27 +527,32 @@ public function should_load() {
 		$should_load = $this->get_option( $option_key );
 
 		if ( '' === $should_load ) {
-
-			// New installs without PayPal Standard enabled don't load it.
-			if ( 'no' === $this->enabled && WC_Install::is_new_install() ) {
-				$should_load = false;
-			} else {
-				$should_load = true;
-			}
+			// Set default `_should_load` to 'yes' on existing stores with PayPal Standard enabled or with existing PayPal Standard orders.
+			$should_load = 'yes' === $this->enabled || $this->has_paypal_orders();
 
 			$this->update_option( $option_key, wc_bool_to_string( $should_load ) );
 		} else {
-			$should_load = wc_string_to_bool( $should_load );
+			// Enabled always takes precedence over the option.
+			$should_load = $this->enabled || wc_string_to_bool( $should_load );
```

Comment:
`$this->enabled` is also a `yes` or `no` string, so we should pass it through `wc_string_to_bool()` too. Otherwise, this will always be true.

Reply:
right, addressed here https://github.com/woocommerce/woocommerce/pull/47205/commits/cc70c2cab04fcc81e766a9d7280a4ad1ab12b20a